NAMESPACE

```diff
@@ -59,15 +59,21 @@ S3method(py_to_r,pandas.core.categorical.Categorical)
 S3method(py_to_r,pandas.core.frame.DataFrame)
 S3method(py_to_r,pandas.core.series.Series)
 S3method(py_to_r,python.builtin.list)
+S3method(py_to_r,scipy.sparse.base.spmatrix)
```

Comment:
Could you update `NEWS.md` as well so that existing users know what the additional coverage includes?